hwilib/devices/btchip/btchip.py

```diff
@@ -332,47 +305,6 @@ def finalizeInput(self, outputAddress, amount, fees, changePath, rawTx=None):
 			result['outputData'] = outputs
 		return result
 
-	def finalizeInputFull(self, outputData):
```

Comment:
what is this?

Reply:
I don't know, but we don't use it :shrug:

I was mostly following the diff of what got added in the first commit that weren't changes since the last update.

Reply:
let me ping Nic on this in case it's something useful :shrug:

Reply:
he says nah, it's a java card api